packages/typespec-go/test/azure-http-specs/azure/resource-manager/methodsubscriptionidgroup/resourcegroupresourceoperations_client_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package methodsubscriptionidgroup_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/stretchr/testify/require"
+
+	"methodsubscriptionidgroup"
+)
+
+var validResource = methodsubscriptionidgroup.ResourceGroupResource{
+	ID:       to.Ptr(fmt.Sprintf("/subscriptions/%s/resourceGroups/test-rg/providers/Azure.ResourceManager.MethodSubscriptionId/resourceGroupResources/rg-resource", subscriptionIdExpected)),
+	Location: to.Ptr("eastus"),
+	Name:     to.Ptr("rg-resource"),
+	Properties: &methodsubscriptionidgroup.ResourceGroupResourceProperties{
+		ProvisioningState:    to.Ptr(methodsubscriptionidgroup.ResourceProvisioningState("Succeeded")),
+		ResourceGroupSetting: to.Ptr(string("test-setting")),
+	},
+	SystemData: &methodsubscriptionidgroup.SystemData{
+		CreatedAt:          to.Ptr(parseTime("2023-01-01T00:00:00.000Z")),
+		CreatedBy:          to.Ptr(string("AzureSDK")),
+		CreatedByType:      to.Ptr(methodsubscriptionidgroup.CreatedByType("User")),
+		LastModifiedAt:     to.Ptr(parseTime("2023-01-01T00:00:00.000Z")),
+		LastModifiedBy:     to.Ptr(string("AzureSDK")),
+		LastModifiedByType: to.Ptr(methodsubscriptionidgroup.CreatedByType("User")),
+	},
+	Type: to.Ptr("Azure.ResourceManager.MethodSubscriptionId/resourceGroupResources"),
+}
+
+func TestResourceGroupResourceOperationsClient_Delete(t *testing.T) {
+	delResp, err := clientFactory.NewResourceGroupResourceOperationsClient().Delete(context.Background(), "test-rg", "sub-resource-1", &methodsubscriptionidgroup.ResourceGroupResourceOperationsClientDeleteOptions{})
+	require.NoError(t, err)
+	require.NotNil(t, delResp)
```

Comment:
`delResp` will never be `nil`.

For methods that don't return a body, you want to do `require.Zero(t, delResp)`.